skbio/alignment/_tabular_msa_repr.py

```diff
@@ -0,0 +1,52 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+
+from skbio.util._metadata_repr import _MetadataReprBuilder
+
+
+class _TabularMSAReprBuilder(_MetadataReprBuilder):
+
+    def _process_header(self):
+        cls_name = self._obj.__class__.__name__
+        if self._obj.dtype is not None:
+            dtype_class_name = self._obj.dtype.__name__
+        else:
+            dtype_class_name = 'No dtype'
+        self._lines.add_line(cls_name + ' <' + dtype_class_name + '>')
```

Comment:
:+1: for no space, but looking into this further, check out this PEP:
https://www.python.org/dev/peps/pep-0484/

It suggests using the mypy strategy of using square braces. This might be more pythonic:
`TabularMSA[DNA]`. (It's actually a pretty cool way of describing higher-kinded types, use `__getitem__` as the constructor instead of a function, might warrant some thought for qiime 2)